cmd/check-spelling/README.md

```diff
@@ -55,6 +55,10 @@ to be accepted as correct. The following lists common examples of such words:
 $ ./kata-spell-check.sh check /path/to/file
 ```
 
+> **Note:** If you have made local edits to the dictionaries, you may need to run
```

Comment:
I think a note makes sense, but could you point it at "Create the master dictionary files" section to avoid duplication?

Reply:
Sure, done. What, you thought I'd actually read the whole document? :wink: